config/manager/controller_manager_config.yaml

```
@@ -9,3 +9,77 @@ webhook:
 leaderElection:
   leaderElect: true
   resourceName: e29e094a.k8s.io
+manager:
+  runtime: containerd
+  otlpEndpoint: ""
+  logLevel: info
+  scheduling:
+    repeatInterval: 24h # to be parsed into time.Duration
+    beginImmediately: true
+  profile:
+    enable: false
+    port: 6060
+  imageJob:
+    successRatio: 1.0 # float; ok with YAML?
+    cleanup:
+      delayOnSuccess: 0s # to be parsed into time.Duration
+      delayOnFailure: 24h
+  pullSecrets: [] # image pull secrets for collector/scanner/eraser
+  nodeFilter:
+    type: exclude # must be either exclude|include
+    selectors:
+      - eraser.sh/cleanup.filter
+components:
+  collector:
+    enable: true
```

should we use 
```suggestion
    enabled: true
```

(applies to others too)

---

SGTM, will do

---

done